src/pages/index.tsx

```diff
@@ -1,16 +1,25 @@
-import Head from "next/head";
-import Image from "next/image";
-import styles from "../../styles/Home.module.css";
+import { useEffect, useState } from "react";
+import styles from "../../styles/Index.module.css";
+
+// This is the index page for your application - since it is a a single page, it will be responsable for hosting all of your components.
```

Comment:
Typos
```suggestion
// This is the index page for your application - since it is a single page, it will be responsible for hosting all of your components.
```

Reply:
Fixed, thanks